tasks/main.yml

```diff
@@ -41,4 +41,18 @@
       - containerd.io
     state: present
     update_cache: yes
-    cache_valid_time: 1800
+
+- name: Install docker-compose
+  when: install_docker_compose | bool
+  block:
+    - name: download docker-compose
+      become: True
+      get_url:
```

Comment:
get_url without a checksum is extremely dangerous, as the file can easily be changed between the point you downloaded it and the point the role is executed (ninja update or even malicious actions). Additionally, is there a strong reason why you do not install it [via apt](https://packages.debian.org/buster/docker-compose) (this is _strongly_ preferred in these repos)?